packages/prisma/src/prisma/schema.prisma

```diff
@@ -32,7 +32,7 @@ model Account {
   ssoType SSOType
   ssoId   String
 
-  user   User   @relation(fields: [userId], references: [id])
+  user   User   @relation(fields: [userId], references: [id], onDelete: Cascade, onUpdate: Cascade)
```

Comment:
Defining `onDelete: Cascade` will ensure that an entity is deleted if, e.g., the corresponding user account is deleted. otherwise, the operation could fail due to db key constraints. more info see https://www.prisma.io/docs/concepts/components/prisma-schema/relations/referential-actions#cascade